### DISCUSSION TITLE
介绍一个Apple Podcast频道

### DISCUSSION THREAD

**Discussion by yibingbingwu** [RoadSideUniversity/ClubHouse2024#28] [General]
最近有机会试了试苹果Podcast上的几个中文节目。推荐这个：
[“南会北教：秘密社会，近代江湖的另一个名字｜江湖丛谈”
](https://podcasts.apple.com/us/podcast/vol-165-%E5%8D%97%E4%BC%9A%E5%8C%97%E6%95%99-%E7%A7%98%E5%AF%86%E7%A4%BE%E4%BC%9A-%E8%BF%91%E4%BB%A3%E6%B1%9F%E6%B9%96%E7%9A%84%E5%8F%A6%E4%B8%80%E4%B8%AA%E5%90%8D%E5%AD%97-%E6%B1%9F%E6%B9%96%E4%B8%9B%E8%B0%88/id1533768416?i=1000633579783)

讲中国民间近代秘密社会的事。江湖存在于国家和家庭之间，在中国，江湖可以幻想，但两边都慎言慎行：国家要么招安，要么打压，要么故意忽略不计。家庭可以看武侠小说，电影，但以家庭为中心的江湖人士却不多。

还一个可能大家也已经知道的：[不明白播客](https://podcasts.apple.com/us/podcast/%E4%B8%8D%E6%98%8E%E7%99%BD%E6%92%AD%E5%AE%A2/id1625856906)

不得不佩服这种有职业信仰的报人，专心为小人物，[忽略的历史](https://podcasts.apple.com/us/podcast/ian-johnson-%E6%B1%9F%E9%9B%AA-%E6%98%9F%E7%81%AB-%E5%AF%B9%E6%8A%97%E5%AE%98%E6%96%B9%E5%8F%99%E4%BA%8B%E7%9A%84%E5%9C%B0%E4%B8%8B%E5%8E%86%E5%8F%B2%E5%AD%A6%E5%AE%B6/id1625856906?i=1000631331184)，做些节目，和官宣抗争

**Comment by ergatea** [comment]
谢谢分享

Bing Wu ***@***.***> 于2023年11月12日周日 11:39写道：

**Comment by Lewisrxliu** [comment]
👍谢谢分享！
我也借此分享一下
共识沙龙 Google podcast 
https://podcasts.google.com/feed/aHR0cHM6Ly9hbmNob3IuZm0vcy8yYjhhNmEwOC9wb2RjYXN0L3Jzcw
共识沙龙讲座分享 YouTube频道：https://www.youtube.com/channel/UCYHXsX3B3LXWnkl4iBsTzog B站：https://space.bilibili.com/555412565
其中秦晖教授分享的历史研究和话题，我觉得非常有深度，值得所有人反复思考
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/3f5d53d2-9fb9-438c-8513-b51393314852)
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/9bba9081-0525-4672-a393-e87039442c72)

**Reply by Lewisrxliu** [reply]
夹边沟2010-王兵 https://youtu.be/NgWi4EydCE4?si=-DfqBv7cuiWvszkS

**Comment by Lewisrxliu** [comment]
推荐 经济学人 Economist Podcast 
https://podcasts.google.com/feed/aHR0cHM6Ly9yc3MuYWNhc3QuY29tL3RoZWVjb25vbWlzdGFsbGF1ZGlv

**Reply by Lewisrxliu** [reply]
本期话题 
https://www.economist.com/leaders/2023/11/16/donald-trump-poses-the-biggest-danger-to-the-world-in-2024
L´Unione Sardahttps://www.unionesarda.it › 世界《经济学人》称唐纳德·特朗普是2024 年最大的全球威胁
国际在线https://news.cri.cn › ...《经济学人》评世界十大危机“特朗普当总统”排第六

**Comment by ergatea** [comment]
https://open.spotify.com/episode/34nNKfwCuHmnJ4MPnVCWXA?si=KkpN20oFTVyYSh0UkCBF9Q

**Reply by Lewisrxliu** [reply]
这个频道不错，spotify久闻大名， 原来还可以做成podcast

**Comment by ergatea** [comment]
这些节目会在多个平台发布

lewis ***@***.***>于2023年11月19日 周日22:04写道：
